Makefile

```diff
@@ -1,10 +1,63 @@
-# Image URL to use all building/pushing image targets
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ROOT_DIR_RELATIVE := .
+
+include $(ROOT_DIR_RELATIVE)/common.mk
+
+# Directories
+TOOLS_DIR := hack/tools
```

Comment:
Let's make this variable repo relative to begin with.

So, `$(REPO_ROOT)/hack/tools`

I see later in the file we prepend the root dir anyway, so this would just be a safer way to specify this directory.

I've not search all throughout yet or anything, but I think we should do the same with other Path vars if not already absolute.